emails/file_expiry.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+    Hr,
+    Link,
+    Section,
+    Text
+} from "@react-email/components";
+import { z } from "zod";
+import { WATcloudEmail } from "./_common/watcloud-email";
+
+const WATcloudOnboardingEmailProps = z.object({
+    name: z.string(),
+    machines: z.array(z.string()),
+    paths: z.array(z.array(z.string())),
+    daysForExpiry: z.string(),
+    deletionDate: z.string()
+});
+
+type WATcloudOnboardingEmailProps = z.infer<typeof WATcloudOnboardingEmailProps>;
+
+export const WATcloudOnboardingEmail = (props: WATcloudOnboardingEmailProps) => {
+    const { name,  machines, paths, daysForExpiry, deletionDate} = WATcloudOnboardingEmailProps.parse(props);
+
+    const previewText = `Welcome to WATcloud! Your access has been updated.`;
+    const expiredPaths = (
+        <div>
+            {machines.map((machine, index) => (
+                <ul key={machine} style={{ fontSize: "14px", lineHeight: "24px" }}>
+                    <li>{machine}</li>
+                    <ul>
+                        {paths[index]?.map((path, pathIndex) => (
+                            <li key={pathIndex}>{path}</li>
+                        ))}
+                    </ul>
+                </ul>
+            ))}
+        </div>
+    )
+
+    return (
+        <WATcloudEmail previewText={previewText}>
+            <Text>Hi {name},</Text>
+            <Text>
+                As per efforts to clean up WATO's file drives, we're looking to delete directories that haven't been used in {daysForExpiry} days.
```

Comment:
Let's emphasize that this will be a routine. E.g.:

```suggestion
        In an ongoing effort to keep WATcloud's file storage efficient and maintain optimal performance, we routinely clean up specific directories that haven't been accessed in the past {daysForExpiry} days.
```